cmd/loom/sendtx.go

```diff
@@ -32,6 +32,8 @@ type deployTxFlags struct {
 	Name       string `json:"name"`
 }
 
+type Policy = ctypes.Policy
```

Comment:
Don't think it's a good idea to re-export a coin type from a module that has nothing to do with it

Reply:
It is being used at the time of unmarshalling inputParams for migrationId 2.

Reply:
You can just use `ctypes.Policy` directly here.

Reply:
That's what I have done :)